src/devices/RDZ100ZB.js

```diff
@@ -0,0 +1,194 @@
+	    systemMode = utils.getKey(legacy.thermostatSystemModes, value, value, Number);
+	}
+	await entity.write('hvacThermostat', {systemMode}, siemensTimeout);
+	return {readAfterWriteTime: 250, state: {system_mode: value}};
+    },
+    convertGet: async (entity, key, meta) => {
+	    await entity.read('hvacThermostat', ['systemMode'], siemensTimeout);
+    }
+};
+
+const tz_local_temperature_calibration = {
+    key: ['local_temperature_calibration'],
+    convertSet: async (entity, key, value, meta) => {
+        utils.assertNumber(value);
+        await entity.write('hvacThermostat', {localTemperatureCalibration: Math.round(value * 10)}, siemensTimeout);
+        return {state: {local_temperature_calibration: value}};
+    },
+    convertGet: async (entity, key, meta) => {
+        await entity.read('hvacThermostat', ['localTemperatureCalibration'], siemensTimeout);
+    }
+};
+
+const tz_occupied_heating_setpoint = {
+    key: ['occupied_heating_setpoint'],
+    options: [exposes.options.thermostat_unit()],
+    convertSet: async (entity, key, value, meta) => {
+        utils.assertNumber(value, key);
+        let result;
+	if (meta.options.thermostat_unit === 'fahrenheit') {
+            result = Math.round(utils.normalizeCelsiusVersionOfFahrenheit(value) * 100);
+        } else {
+            result = Number((Math.round(Number((value * 2).toFixed(1))) / 2).toFixed(1)) * 100;
+        }
+        const occupiedHeatingSetpoint = result;
+        await entity.write('hvacThermostat', {occupiedHeatingSetpoint}, siemensTimeout);
+        return {state: {occupied_heating_setpoint: value}};
+    },
+    convertGet: async (entity, key, meta) => {
+        await entity.read('hvacThermostat', ['occupiedHeatingSetpoint'], siemensTimeout);
+    }
+};
+
+const tz_unoccupied_heating_setpoint = {
+    key: ['unoccupied_heating_setpoint'],
+    options: [exposes.options.thermostat_unit()],
+    convertSet: async (entity, key, value, meta) => {
+        utils.assertNumber(value, key);
+        let result;
+        if (meta.options.thermostat_unit === 'fahrenheit') {
+            result = Math.round(utils.normalizeCelsiusVersionOfFahrenheit(value) * 100);
+        } else {
+            result = Number((Math.round(Number((value * 2).toFixed(1))) / 2).toFixed(1)) * 100;
+        }
+        const unoccupiedHeatingSetpoint = result;
+        await entity.write('hvacThermostat', {unoccupiedHeatingSetpoint});
+        return {state: {unoccupied_heating_setpoint: value}, siemensTimeout};
+    },
+    convertGet: async (entity, key, meta) => {
+        await entity.read('hvacThermostat', ['unoccupiedHeatingSetpoint'], siemensTimeout);
+    }
+};
+
+const definition = {
+    zigbeeModel: ['RDZ100'],
+    model: 'RDZ100',
+    vendor: 'Siemens',
+    description: 'Siemens RDZ100 Wireless Thermostat',
+    fromZigbee: [
+        siemensFromZigbee,
+    ],
+    toZigbee: [
+	tz_local_temperature,
+	tz_control_sequence_of_operation,
+	tz_system_mode,
```

Comment:
Can you switch to the standard converters, e.g. `tz.thermostat_system_mode`?

Reply:
What about the timeout then? That device accept command one per minute, when i tried timeout value in top layer meta it does nothing and i do not see possibility to set it otherwise. Is there some better way to wait for incomming event first? Btw also would it be possible to move description of all generic stuff to optional properties with default value of current texts? (I can do that in different pull request)

Reply:
> Is there some better way to wait for incomming event first? 

I've replied on the discussion

> Btw also would it be possible to move description of all generic stuff to optional properties with default value of current texts? (I can do that in different pull request)

Do you mean this one?

```
expose.push(e.enum('control_sequence_of_operation', ea.STATE_SET, Object.values(siemensOperationModes)).withDescription('Operation mode'));
```